project/SparkBuild.scala

```diff
@@ -654,7 +654,10 @@ object DependencyOverrides {
  */
 object ExcludedDependencies {
   lazy val settings = Seq(
-    libraryDependencies ~= { libs => libs.filterNot(_.name == "groovy-all") }
+    libraryDependencies ~= { libs => libs.filterNot(_.name == "groovy-all") },
+    excludeDependencies ++= Seq(
+      ExclusionRule("javax.ws.rs", "jsr311-api")
```

Comment:
This is quite surprising. I thought SBT respects pom.xml and this file is just some extra dep exclusions for SBT.

Reply:
there are 40 + artifacts in sbt than maven...

Reply:
scala> iamsbt.length
res30: Int = 323

scala> iammvn.size
res31: Int = 279